clinica/utils/inputs.py

```diff
@@ -345,12 +345,10 @@ def find_sub_ses_pattern_path(
             )
             results.append(selected)
         else:
-            error_str = f"\t*  ({subject} | {session}): More than 1 file found:\n"
-            for found_file in current_glob_found:
-                error_str += f"\t\t{found_file}\n"
-            error_encountered.append(error_str)
+            # todo : RQ : now we do not know why the invalid files are considered invalid
+            error_encountered += [(subject, session)]
```

Comment:
Small suggestion: we could use named tuples here to make things easier to read.

It would look more or less like that (maybe there is a better name than `ProblematicSubjectSession` ?? I was going for `ClinicaFileReaderError` but it is misleading as not an error per se):

```python
>>> from collections import namedtuple
>>> ProblematicSubjectSession = namedtuple("ProblematicSubjectSession", ["subject", "session"])
>>> err = ProblematicSubjectSession("sub-01", "ses-M006")
>>> err
ProblematicSubjectSession(subject='sub-01', session='ses-M006')
```

This would avoid doing `err[0]` or `err[1]` later in the code, and instead:
```python
>>> err.subject
'sub-01'
>>> err.session
'ses-M006'
```

Also, this would provide a richer type hinting, since we could have:

```python
def func(errors: Iterable[ProblematicSubjectSession]):
    pass
```

We could also add the actual reason or the message to the named tuple in order to keep this information for error formatting. I'm not sure it's worth it though as your solution already points the user to the problematic subjects and sessions, which seems fine to me.

Reply:
Thanks for the input !! I am just using subject and session for now but that can be modified later if something feels missing